2024/ig-security.html

```diff
@@ -160,7 +160,7 @@ <h2>Scope</h2>
         <p>The Security Interest Group (SING) develops and documents guidelines, patterns, processes, and best practices for addressing security issues in Web standards.</p>
 	<p>SING supports, promotes, and structures the threat modeling for web standards and technologies. This approach can be used, along with other groups, for threats of different types such as security, privacy, and other kinds of harm. Threat modeling is a joint activity with threat experts and groups developing technology or other documentation. It can be used to get an understanding of the impact of the technology and guide its development, as well as to write Security Considerations sections.</p>
         <p>SING provides "<a href="https://www.w3.org/Guide/documentreview/">horizontal review</a>", offering groups on-request guidance on security issues and mitigations specific to their technologies. SING aims to offer this review as early in the technology development lifecycle as requested, observing that early feedback is often more helpful. SING may also seek out technologies that benefit from earlier security reviews and conduct such reviews on its initiative.</p>
-        <p>SING incubates standards work on security issues by collecting requirements, prototyping, and/or initiating the work within the IG and recommending that the W3C move the work into other groups when appropriate.</p>
+        <p>SING explores security issues on standards work by collecting requirements, prototyping, and/or developing tests within the IG and recommending that the W3C move the work into other groups when appropriate.</p>
```

Comment:
I think this was better as "explores standards work on security issues" or s/on/for/. It's the standards work that the IG will eventually propose, not the security issues.

Reply:
@jyasskin "explores standards work" sounds far too broad for an IG, IGs are not supposed to pursue actual standards work at all, in any way, whether incubating or exploring.

Reply:
Other IGs have:
* "[Identify and communicate standardization opportunities for W3C](https://www.w3.org/2022/11/chinese-web-ig-charter.html#scope:~:text=Identify%20and%20communicate%20standardization%20opportunities%20for%20W3C)"
* "[the primary focus of the group is expected to be on discovering and defining standardization items](https://www.w3.org/2023/12/web-networks-charter.html#scope:~:text=the%20primary%20focus%20of%20the%20group%20is%20expected%20to%20be%20on%20discovering%20and%20defining%20standardization%20items)"
* "[identifying appropriate standardization targets](https://www.w3.org/2024/04/wot-ig-2024.html#:~:text=identifying%20appropriate%20standardization%20targets)"
* "[Identify standardization opportunities](https://www.w3.org/securepay/charter-2023.html#activities:~:text=Identify%20standardization%20opportunities)"
* "[Identification and prioritization of gaps in Web standards for Smart City use cases](https://www.w3.org/2024/06/smart-cities/#scope:~:text=Identification%20and%20prioritization%20of%20gaps%20in%20Web%20standards%20for%20Smart%20City%20use%20cases)"

So ... maybe "identifies" is a better word than "explores"?

```suggestion
        <p>SING identifies standardization work on security issues by collecting requirements, prototyping, and/or developing tests within the IG and recommending that the W3C move the work into other groups when appropriate.</p>
```

I don't strongly object to @tantek's change; I'm just not sure what a "security issue on standards work" is, or how one explores a security issue by collecting requirements, etc.